docker/wait-for-db.sh

```diff
@@ -18,15 +18,18 @@ set -eu
 
 host="${1}"
 port="${2}"
+timeout="${3:-20}"
+NEXT_WAIT_TIME=0
 
-until PGPASSWORD="${POSTGRES_PASSWORD}" psql \
+until [ PGPASSWORD="${POSTGRES_PASSWORD}" psql \
```

Comment:
_Minor_: `[[ ... ]]` is preferred over `[`, see [Google Shell Style Guide](https://google.github.io/styleguide/shell.xml?showone=Test,_%5B_and_%5B%5B#Test,_%5B_and_%5B%5B).